frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -4,15 +4,15 @@ import { appHost } from '../protractor.conf';
 import * as loginView from '../views/login.view';
 
 const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 4;
```

Comment:
I wouldn't change this. If it's taking more than 3 minutes for login to work, that's a problem. I think it is better for the tests to fail in that case.

It might be better if we reduce this value honestly now that the OLM tests have been moved to Cypress. IIRC, we needed a long wait because create the catalog source in the OLM tests took a while.